cookiecutter.json

```diff
@@ -1,15 +1,32 @@
 {
-  "project_name": "My Site",
-  "project_shortname": "{{ cookiecutter.project_name | slugify() }}",
-  "project_site": "{{ cookiecutter.project_shortname }}.com",
-  "github_repo": "{{ cookiecutter.project_shortname }}/{{ cookiecutter.project_shortname }}",
-  "description": "{{ cookiecutter.project_name }} InvenioRDM Instance",
-  "author_name": "CERN",
-  "author_email": "info@{{ cookiecutter.project_site }}",
-  "year": "{% now 'local', '%Y' %}",
-  "python_version": ["3.8", "3.7", "3.9"],
-  "database": ["postgresql"],
-  "search": ["opensearch2", "opensearch1", "elasticsearch7"],
-  "file_storage": ["local", "S3"],
-  "development_tools": ["yes", "no"]
-}
+    "project_name": "My Site",
```

Comment:
moderate/major: be careful seems your editor is not respecting `.editorconfig`. So you might introduce changes where there are not, also harder to spot the actual changes 😅. In cookiecutter it says indent 2 for json files. See [here](https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/.editorconfig#L32).